nf_upf/Dockerfile

```diff
@@ -9,7 +9,10 @@ ENV DEBIAN_FRONTEND noninteractive
 ARG DEBUG_TOOLS
 
 # Install debug tools ~ 100MB (if DEBUG_TOOLS is set to true)
-RUN if [ "$DEBUG_TOOLS" = "true" ] ; then apt-get update && apt-get install -y vim strace net-tools iputils-ping curl netcat ; fi
+RUN if [ "$DEBUG_TOOLS" = "true" ] ; then apt-get update && apt-get install -y golang-go git vim strace net-tools iputils-ping curl netcat && \
+    git clone https://github.com/free5gc/go-gtp5gnl.git && mkdir "go-gtp5gnl/bin" && \
```

Comment:
move go-gtp5gnl tool build to base Dockerfile and copy it from base

Reply:
If I move this build action to the base Dockerfile, I think it can't be optional due to we won't use the base image directly (Won't use `DEBUG_TOOLS`).